frontend/src/components/regions/UserFeedbackChart/Presentation.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+} from "recharts";
+import orange from "@material-ui/core/colors/orange";
+import green from "@material-ui/core/colors/green";
+import red from "@material-ui/core/colors/red";
+import blue from "@material-ui/core/colors/blue";

```

Comment:
This might need to change later on, for the sake of DRY code

Reply:
Why cant we change this now than later?